### ISSUE TITLE
screen tearing

### ISSUE THREAD

**Issue by trs79** [EMGD-Community/intel-binaries-linux#38]
I'm using EMGD 1.18 on Ubuntu 14.04 32bit, and while 2D and 3D acceleration is working, there is significant tearing when moving graphics horizontally. Is this a known issue? Thanks.

**Comment by thopiekar** [comment]
I have nothing to compare (maybe you can make a video), but it is possible that you'll need to add some options to the xorg.conf.
More than a year ago I've made some additions to the screen setup there, so please take a look at the user-guide hosted in this repository as pdf. In my case (Asus T91) it looks like this: https://github.com/EMGD-Community/intel-binaries-linux/blob/master/config/xorg.conf/ASUS_T91.conf#L47

I'm glad to hear it is working so far :+1: 
Regards

**Comment by trs79** [comment]
Thanks for the response. There is a screenshot at this link: http://www.iwillfolo.com/finally-nvidia-tearing-issue-fix-found/ that shows an example of the same issue I'm having, when you drag a window horizontally across the screen it won't move smoothly but will have portions that are off.

I have a working xorg.conf that I got from another distro specifically for the device I'm using (an OpenFrame). I'm hesitant to try changing any values as it would probably prevent x from starting.

I checked the xorg log, end it says "TearFB" is enabled (which is supposed to prevent tearing, but seems to have no effect for me). Do you notice any tearing on your system? I'm using Ubuntu 14.04 32 bit with kernel 3.18.2

**Comment by thopiekar** [comment]
Well, if it is enabled, then you probably got the best performance you can
get.
Any other tweaks can only be done by modifying the values of the EMGD
driver.
Am 14.01.2016 5:18 vorm. schrieb "trs79" <notifications@github.com>:

**Comment by trs79** [comment]
When you say modifying the values of the driver, do you mean parameters passed to the module? I'm just curious, do you happen to notice any tearing? I'm wondering if it's just me, or if this is normal behavior for EMGD, thanks.

**Comment by thopiekar** [comment]
Yes, those values are well documented in the user guide. You can make a copy of your current xorg.conf and play with these values using the guide.

Yes, there is tearing (without looking whether TearFB is on), but so far I didn't took care about it. I'm happy that this hardware is even still working. I mean on every Ubuntu release I have to take days to get it working again, because of changes in newer kernel versions and changes to the GCC compiler. So having 3D on my old netbook is more than great. Thats why I wasn't playing on minimizing tearing.

So again, just take the user guide I uploaded here, it is the only option how to minimize the problem :sweat_smile:

**Comment by trs79** [comment]
Thanks for all your work on making the driver work, I too am excited it's working on my old Joggler device. For now I'll just have to accept the tearing I guess
